src/renderer/components/poolActionsHistory/PoolActionsHistory.helper.test.ts

```diff
@@ -128,4 +128,53 @@ describe('PoolActionsHistory.helper', () => {
       ).toBeTruthy()
     })
   })
+  describe('getRowKey', () => {
+    it('should return correct id if exists', () => {
+      expect(
+        getRowKey({
+          ...defaultPoolAction,
+          in: [
+            {
+              ...defaultTx,
+              txID: 'inId'
+            }
+          ]
+        })
+      ).toEqual('inId')
+
+      expect(
+        getRowKey({
+          ...defaultPoolAction,
+          out: [
+            {
+              ...defaultTx,
+              txID: 'outId'
+            }
+          ]
+        })
+      ).toEqual('outId')
+
+      expect(
+        getRowKey({
+          ...defaultPoolAction,
+          in: [
+            {
+              ...defaultTx,
+              txID: 'inId'
+            }
+          ],
+          out: [
+            {
+              ...defaultTx,
+              txID: 'outId'
+            }
+          ]
+        })
+      ).toEqual('inId')
+    })
+
+    it('should return default value in case there is no txId (`dateTimrStamp-action.type`)', () => {
+      expect(getRowKey(defaultPoolAction)).toEqual(`${new Date(0)}-SWAP`)
```

Comment:
[question] Why `new Date(0)` (which is always the same) and not just `swap` or anything else?

Reply:
[nitpick] Ah, it's because you use `action.date.toString()}-${action.type}` later. So just add a short note to understand why to use `new Date(0)`  ;)

Reply:
hm. it's described at the test-case name. no ? 🤔